vue/components/molecules/restrictions-alert/restrictions-alert.vue

```diff
@@ -77,14 +77,20 @@ export const RestrictionsAlert = {
             visible: false
         };
     },
-    mounted: function() {
-        this.$bus.bind("restrictions", (changes, newPart) => {
+    created: function() {
+        this.onRestrictions = this.$bus.bind("restrictions", (changes, newPart) => {
             this.visible = changes.length > 0;
         });
+
+        this.onUndo = this.$bus.bind("undo", () => this.close());

```

Comment:
Explain in the `Decisions` why this was the approach, in spite of what was discussed in the ticket.